manifests/resource-topology-exporter-ds.yaml

```diff
@@ -48,11 +39,10 @@ spec:
       labels:
         name: resource-topology
     spec:
-      hostNetwork: true
-      serviceAccountName: rte-account
+      serviceAccountName: rte
       containers:
       - name: resource-topology-exporter-container
-        image: quay.io/swsehgal/resource-topology-exporter:latest
+        image: ${RTE_CONTAINER_IMAGE}
```

Comment:
Maybe like NFD we could also have a template files where ${RTE_CONTAINER_IMAGE} replaced by appropriate name everytime `make image` is executed? Of course not needed here but is a nice to have, could be done in a separate PR.

Reply:
yes, I prefer to postpone because is not yet clear where the prebuilt image is going to sit, and I think we should avoid personal repos if possible.